eslint.config.mjs

```diff
@@ -37,6 +53,12 @@ export default [
       '@typescript-eslint/no-empty-object-type': 'off',
     },
   },
+  {
+    files: ['**/action-runner.ts'],
+    rules: {
+      '@typescript-eslint/no-unused-vars': 'off',
+    },
+  },
   {
     files: [...tsFileExtensions, ...jsFileExtensions, '**/*.tsx'],
```

Comment:
dont think we need these, it totally defeats the purpose of pre commit hooks validations